src/components/Hero/Hero.module.scss

```diff
@@ -85,3 +85,49 @@
   height: 100%;
   background-color: rgba(0, 0, 0, 0.3);
 }
+
+@media (max-width: 600px) {
+  .heroFooter {
+    min-width: 100vw;
+    font-size: 20vw;
+    gap: 1vh;
+    right: 0;
+    flex-direction: column;
+    bottom: 14;
+    padding: 0 1vw 2vh 0;
+  }
+
+  .heroType {
```

Comment:
the naming isn't clear to me. also this file is named hero so we can leave out the "hero" prefix. For this, maybe `exerciseTypeTitle` or something